internal/xds/translator/listener.go

```diff
@@ -146,36 +147,6 @@ func originalIPDetectionExtensions(clientIPDetection *ir.ClientIPDetectionSettin
 	return extensionConfig
 }
 
-func setAddressByIPFamily(socketAddress *corev3.SocketAddress, ipFamily *ir.IPFamily, port uint32) []*listenerv3.AdditionalAddress {
```

Comment:
why does this function need to change ?

Reply:
Before this function change `socketAddress` and return `additionalAddress` in the same time.